### ISSUE TITLE
Provide a working example of using vue-fontawesome

### ISSUE THREAD

**Issue by adamdehaven** [Maronato/vue-toastification#236]
I've tried importing FontAwesome as components and other methods and can't figure out how to utilize custom [vue-fontawesome](https://github.com/FortAwesome/vue-fontawesome) icons with the `vue-toastification` library. 

With FontAwesome being hugely popular, it would be super-awesome of you to show a working example of how to get this working 🎉 

*If there is already a working example somewhere that I somehow missed, please point me to it and I'll humbly close this issue in shame*

**Comment by adamdehaven** [comment]
Can someone just provide a working example...?

**Comment by LovelyAndy** [comment]
Any updates with this? I'm able to render the icons from my icon pack (IconPark), but I cannot figure out how to change the size of the icon at all. A working example would be really helpful, since the `_icon.scss` page is a bit hard to parse. 

Cheers on the great library!

**Comment by Maronato** [comment]
Hello!
Here is a working example: https://stackblitz.com/edit/node-etdpeu?file=src%2Fmain.ts

![CleanShot 2021-11-04 at 17 33 13](https://user-images.githubusercontent.com/19658460/140415661-95402a4f-d720-4ee9-b83f-1ad490388557.png)

For more information, see the [docs](https://github.com/Maronato/vue-toastification#using-custom-icons)
